.jenkinsci/release-build.groovy

```diff
@@ -105,7 +106,24 @@ def doReleaseBuild() {
       }
     }
   }
-
+  if (GIT_LOCAL_BRANCH == 'master' && checkTag == 0) {
```

Comment:
Please, keep the existing functionality of pushing Docker images that don't have Git tag attached in the code. I.e., push Docker image with `master` tag if tag is missing but ALSO push an image with `<tag>` tag if the tag is present.
Consider converting dashes ('-') to underscores ('_') in tag names